CHANGELOG.md

```diff
@@ -326,10 +326,18 @@ Released with 1.0.0-beta.37 code base.
 - Rename `web3-eth2-base` to `web3-eth2-core` and `web3-eth2-beacon` to `web3-eth2-beaconchain`
 - Bump `ts-node` from version `^8.10.2` to `^9.0.0`
 - Ran `npm audit fix` which fixed 4 vulnerabilities
-- Correct `web3-eth2-beaconchain` type declarations
+- Correct `web3-eth2-beaconchain` type declarations (#3859) and (#3865)
+- Move interfaces `IBaseAPISchema` and `IBaseAPIMethodSchema` to `index.d.ts` for `web3-eth2-core`
```

Comment:
Can you makr the PR number for all these, please!

Reply:
Updated them :slightly_smiling_face: